packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -121,11 +121,14 @@ class ChartState extends State<Chart> with AutoDisposeMixin {
   }
 }
 
+/// Used to track current and previous x,y position and where the bottom
+/// of the Y is located (for non stacked traces yBottom is 0).
 class Point {
-  Point(this.x, this.y);
+  Point(this.x, this.y, this.yBottom);
```

Comment:
nit: I'd suggest making yBottom a required named parameter to improve the readability of code using this.

Reply:
```
/// Used to track current and previous x,y position. The parameter yBase
/// is for stacked traces. If the Trace is stacked then yBase is the previous
/// trace's Y position. If a trace is not stacked then yBase is always 0.
class PointAndBase {
  PointAndBase(
    this.x,
    this.y, {
    double yBase,
  }) : base = yBase;

  final double x;
  final double y;
  final double base;
}

```